core/events/tests/test_classic_integration.py

```diff
@@ -218,3 +232,39 @@ def test_classic_workflow(self):
                              "Submit time is set.")
             self.assertEqual(len(stack), 11,
                              "Eleven commands have been executed in total.")
+    
+    
+    def test_unicode_submitter(self):
+        """Submitter proceeds through workflow in a linear fashion."""
+        submitter = self.unicode_submitter
+        metadata = [
+            ('title', '优秀的称号'),
+            ('abstract', "当我有一天正在上学的时候"),
+            ('comments', '5页2龟鸠'),
+            ('report_num', 'asdf1234'),
+            ('doi', '10.01234/56789'),
+            ('journal_ref', 'Foo Rev 1, 2 (1903)')
+        ]
+        authors = [events.Author(
```

Comment:
In ARXIVNG-458, let's let the caller pass "raw" data (not require domain class instances).